zuul.d/edpm.yaml

```diff
@@ -7,7 +7,7 @@
     abstract: true
     parent: base-simple-crc
     vars:
-      crc_parameters: "--memory 21000 --disk-size 120 --cpus 8"
+      crc_parameters: "--memory 24000 --disk-size 120 --cpus 8"
```

Comment:
Please change the parameters to `--memory 22000 --disk-size 120 --cpus 10` It will help to keep 2 GB/ 2 Vcpus for host and for baremetal 8 GB ram and 4 Vcpus.

Reply:
let's see how your iteration goes :)